test-harness/index.ts

```diff
@@ -43,60 +45,79 @@ describe('harness', () => {
 
     const testText = `${file}${os.EOL}${parsed.test}`;
 
-    test(testText, done => {
+    test(testText, async done => {
       const [command, ...args] = parsed.command.split(/ +/).map(t => t.trim());
-      const serverArgs = parsed.server.split(/ +/).map(t => t.trim().replace('${document}', tmpFileHandle.name));
-      prismMockProcessHandle = spawn(path.join(__dirname, '../cli-binaries/prism-cli'), serverArgs);
-
-      prismMockProcessHandle.stdout.pipe(split2()).on('data', (line: string) => {
-        if (line.includes('Prism is listening')) {
-          const clientCommandHandle = spawnSync(command, args, {
-            shell: true,
-            encoding: 'utf8',
-            windowsVerbatimArguments: false,
-          });
-          const output: any = parseResponse(clientCommandHandle.stdout.trim());
-          const expected: any = parseResponse((parsed.expect || parsed.expectLoose).trim());
 
-          const isXml = xmlValidator.test(
-            get(output, ['header', 'content-type'], ''),
-            expected.body
-          );
+      const prismMockProcessHandle = await startPrism(parsed.server, tmpFileHandle.name);
```

Comment:
…per the comment above, we should modify the `startPrism` function to take an additional done callback to continue the flow later.

Reply:
Done 👍